boa/src/builtins/console.rs

```diff
@@ -32,11 +32,36 @@ pub fn error(_: &Value, args: &[Value], _: &mut Interpreter) -> ResultValue {
     Ok(Gc::new(ValueData::Undefined))
 }
 
+/// Print a javascript value to the standard error if first argument evaluates to false or there were no arguments
+/// 
+/// https://console.spec.whatwg.org/#assert
```

Comment:
Actually, I meant something like this:
```suggestion
/// `console.assert(condition, ...data)`
///
/// Prints a JavaScript value to the standard error if first argument evaluates to `false` or there
/// were no arguments.
/// 
/// More information: <https://console.spec.whatwg.org/#assert>
```

Reply:
Ok, I will write docs like that in the future.

One more question: should I make one pr per implemented function or wait until everything will be done?